packages/io-services-cms-models/src/lib/fsm/store.cosmos.ts

```diff
@@ -115,7 +119,12 @@ export const createCosmosStore = <
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
       ({ last_update, version, ...valueToSave }) =>
         TE.tryCatch(
-          () => container.items.upsert({ ...valueToSave, id }),
+          () =>
+            container.items.upsert({
+              ...valueToSave,
+              id,
+              modified_at: unixTimestamp(),
```

Comment:
**issue:** there is a case for which the `modified_at` field should not be overwritten with current timestamp!

When synching from legacy, the field modified_at will be filled with legacy timestamp and that timestamp should be saved as `modified_at`, not the current one!

Reply:
93edf2f6f8de40684f88f812d787b6a44a74a775